main.py

```diff
@@ -15,11 +15,13 @@
     parser.add_argument(
```

Comment:
Log level should be taken from the same config file as the fybrik control plane, shouldn't it?